imports/ui/components/password/client/password.tests.js

```diff
@@ -0,0 +1,72 @@
+import {
+    name as Password
+} from '../password';
+
+import 'angular-mocks';
+
+import {
+    Meteor
+} from 'meteor/meteor';
+import {
+    Accounts
+} from 'meteor/accounts-base';
+
+import {
+    sinon
+} from 'meteor/practicalmeteor:sinon';
+
+describe('Password', function() {
+    spies.restoreAll();
+
+    beforeEach(function(done) {
+        window.module(Password);
+        done();
+    });
+
+    describe('controller', function() {
+        let controller;
+
+        beforeEach(function(done) {
+            inject(function($rootScope, $componentController) {
+                controller = $componentController(Password, {
+                    $scope: $rootScope.$new(true)
+                });
+            });
+            done();
+        });
+
+        it('should have credentials all empty by default', function(done) {
+            expect(controller.credentials).to.be.deep.equal({
+                email: ''
+            });
+            done();
+        });
+
+        it('should have error empty by default', function(done) {
+            expect(controller.error).to.be.equal('');
+            done();
+        });
+
+        describe('reset()', function() {
+            let validEmail = 'validEmail';
+
+            afterEach(function(done) {
+                if (spies.forgot)
+                    spies.forgot.restore();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'spies'.](https://www.codacy.com/app/ddspog/MAM-Seed/file/2729684822/issues/source?bid=3340041&fileBranchId=3346427#l55)